driver/tab_completion.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import importlib.util
+from pathlib import Path
+import subprocess
+import sys
+
+from . import util
+
+try:
+    import argcomplete
+    HAS_ARGCOMPLETE = True
+except ImportError:
+    HAS_ARGCOMPLETE = False
+
+
+def complete_build_arg(prefix, parsed_args, **kwargs):
+    if parsed_args.debug:
+        argcomplete.warn("The option --debug is an alias for --build=debug. Do no specify both --debug and --build.")
+        exit(1)
```

Comment:
The "argcomplete.warn + exit" combo occurs at least three times, so I think we should encapsulate it in a function. The exit code shouldn't be 1; see "https://www.fast-downward.org/ExitCodes". The closest match is perhaps DRIVER_INPUT_ERRROR, which should already be defined as a symbolic constant somewhere.